src/api/v1beta1/properties.go

```diff
@@ -241,7 +242,7 @@ func (dynakube DynaKube) CodeModulesVersion() string {
 		codeModulesImage := dynakube.CodeModulesImage()
 		return strings.Split(codeModulesImage, ":")[1]
 	}
-	if dynakube.Version() != "" {
+	if dynakube.Version() != "" && !dynakube.CloudNativeFullstackMode() {
```

Comment:
Nitpick: the check if version is empty or not could be a property to enhance readability

```suggestion
	if dynakube.HasVersion() && !dynakube.CloudNativeFullstackMode() {
```